--- a/source3/script/installmodules.sh.org	2015-02-22 15:11:32.000000000 +0100
+++ b/source3/script/installmodules.sh	2019-03-09 10:27:49.274989036 +0100
@@ -29,11 +29,11 @@
 for p in $*; do
  p2=`basename $p`
  name=`echo $p2 | sed -es/${mtype}_//`
- echo Preserving old module as $DESTDIR/$LIBDIR/$name.old
- if [ -f $DESTDIR/$LIBDIR/$name ]; then
-   rm -f $DESTDIR/$LIBDIR/$name.old
-   mv $DESTDIR/$LIBDIR/$name $DESTDIR/$LIBDIR/$name.old
- fi
+ #echo Preserving old module as $DESTDIR/$LIBDIR/$name.old
+ #if [ -f $DESTDIR/$LIBDIR/$name ]; then
+ #  rm -f $DESTDIR/$LIBDIR/$name.old
+ #  mv $DESTDIR/$LIBDIR/$name $DESTDIR/$LIBDIR/$name.old
+ #fi
  echo Installing $p as $DESTDIR/$LIBDIR/$name
  cp -f $p $DESTDIR/$LIBDIR/$name
  chmod $INSTALLPERMS $DESTDIR/$LIBDIR/$name
